### ISSUE TITLE
Update Version on cn Cloud

### ISSUE THREAD

**Issue by genhack** [DejavuMoe/Hackintosh-OMEN-HP-Laptop-15-dc0xxx#2]
hello friend can you download this and post on github 

https://cloud.189.cn/t/um2IJfzeUjIn  Pasw: l5qk 

Thanks!

**Comment by DejavuMoe** [comment]
Thanks for sharing, I was preparing for my final exams a few days ago so I'm just now seeing the news. I'm testing this EFI and if it's available I'll do some minor fixes as appropriate. If you can, can you tell me the source of this EFI?

**Comment by DejavuMoe** [comment]
> hello friend can you download this and post on github
> 
> https://cloud.189.cn/t/um2IJfzeUjIn Pasw: l5qk
> 
> Thanks!


I spent the afternoon yesterday formatting my hard drive to try that EFI on my OMEN device and had issues with the laptop keyboard and touchpad freezing completely, so I updated the kext for the touchpad and keyboard devices. because it's hot 🔥 and the daily use laptop temperature is over fifty degrees both sides of the fan are not spinning and it seems to trigger the fan to spin the temperature wall is high?

This is the end of my attempts, so far EFI is working well that's enough. I will go back to Linux or Windows, thanks for sharing 🤷♂️ I wonder if you are the original author of EFI?

![Snipaste_2021-07-12_10-17-44](https://user-images.githubusercontent.com/57177121/125221443-ca612e80-e2fa-11eb-918b-ae20609ad2a8.png)
